src/api/spec/models/notification_spec.rb

```diff
@@ -33,4 +33,43 @@
 
     it { expect(test_group.notifications).to include(notification) }
   end
+
+  describe '#user_active?' do
+    subject { rss_notification.user_active? }
+
+    let(:rss_notification) { create(:rss_notification, subscriber: test_user) }
+
+    context 'when subscriber is away' do
+      let(:test_user) { create(:dead_user, login: 'foo') }
+
+      it { expect(subject).to be_falsey }
+    end
+
+    context 'when subscribe logged in recently' do
+      let(:test_user) { create(:confirmed_user, login: 'foo') }
+
+      it { expect(subject).to be_truthy }
+    end
+  end
+
+  describe '#any_user_in_group_active?' do
+    subject { rss_notification.any_user_in_group_active? }
+
+    let(:rss_notification) { create(:rss_notification, subscriber: test_group) }
+    let(:test_group) { create(:group) }
+
+    before do
+      test_group.add_user(test_user)
+    end
+
+    context 'no active user in the group' do
+      let!(:test_user) { create(:dead_user, login: 'foo') }
+      it { expect(subject).to be_falsey }
```

Comment:
Nitpick: We usually insert the `let` lines from the `it` lines with an empty line, to clarify what is a variable declaration and what is a test. It didn't happen in the previous '#user_active?' section, but it happens in this line and also in the next context.

Reply:
done